lib/command/src/Obelisk/Command/Run.hs

```diff
@@ -88,21 +86,44 @@ run = do
       , "Frontend.frontend"
       ]
 
+-- | Nix syntax requires relative paths to be prefixed by @./@ or
+-- @../@. This will make a 'FilePath' can be embedded in a Nix
```

Comment:
```suggestion
-- @../@. This will make a 'FilePath' that can be embedded in a Nix
```

Reply:
Probably shouldn't block merge at this point

Reply:
Fixed, thanks.